frontend-student/src/components/dashboard/DashboardSummary.jsx

```
@@ -0,0 +1,22 @@
+        <h2 className='text-xl font-semibold text-gray-700'>
+          Overall Learning Progress
+        </h2>
+        <span className='text-2xl font-bold text-blue-600'>
+          {Math.round(overallProgress)}%
+        </span>
+      </div>
+      <div className='w-full bg-gray-200 rounded-full h-4'>
+        <div
+          className='bg-blue-600 h-4 rounded-full transition-all duration-500'
+          style={{ width: `${overallProgress}%` }}
+        ></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)